### ISSUE TITLE
Feature: Personal Dictionary Synchronization

### ISSUE THREAD

**Issue by frederikb96** [Clemens-E/obsidian-languagetool-plugin#73]
I would really like to be able to update my personal dictionary of my language tool account via the Obsidian Plugin.
- Currently, words added to the personal dictionary in my language tool account are already recognized by the Obsidian Plugin, but if I add new ones, they are not uploaded.
- In #41, they say, that new entries would be stored in the Obsidian dictionary but not via the language tool account

According to the [documentation of the language tool API](https://languagetool.org/http-api/#!/default/post_words_add), it is possible to add new words to the dictionary

Thanks :)

**Comment by berot3** [comment]
+1

But, are you having premium? 

I was wondering whether it is possible to get an API key with non-premium accounts? 

If premium: maybe the API automatically uses you personal dict? But yes, for premium users it would be nice to add words to your own LT-dict.

**Comment by frederikb96** [comment]
True, forgot to mention that I have premium. Yeah, the dictionary might be automatically accessed in that case.
